securedrop/tests/test_journalist_api.py

```diff
@@ -1055,23 +1054,6 @@ def test_reply_download_generates_checksum(journalist_app,
     assert not mock_add_checksum.called
 
 
-def test_revoke_token(journalist_app, test_journo, journalist_api_token):
```

Comment:
This test should be replaced with a similar one but leveraging the new logout mechanism.